infrastructure/cdn-in-a-box/traffic_ops/trafficops-init.sh

```diff
@@ -102,7 +102,7 @@ load_data_from() {
 load_data_from /traffic_ops_data
 
 # Copy the free MaxMind GeoLite DB to TrafficOps public directory
-tar -C /var/tmp -zxpvf /GeoLite2-City.tar.gz
-geo_dir=$(find /var/tmp -maxdepth 1 -type d -name GeoLite2-City\*)
-gzip -c "$geo_dir/GeoLite2-City.mmdb" > "$TO_DIR/public/GeoLite2-City.mmdb.gz"
-chown trafops:trafops "$TO_DIR/public/GeoLite2-City.mmdb.gz"
+# tar -C /var/tmp -zxpvf /GeoLite2-City.tar.gz
```

Comment:
should we just remove this code instead of commenting out?

Reply:
yes